sdk/couchbase/src/error.rs

```diff
@@ -76,6 +76,10 @@ impl Error {
             collection_name,
         );
 
+        if let Some(source) = e.source() {
+            extended_context = extended_context.with_source_message(source.to_string());
```

Comment:
What's the difference between how source was previously being specified and how it is now being included in extended_context, from a user perspective?

Reply:
Previously it was totally missing, so the user wouldn't have seen this detail at all.